daemon/invoice.h

```diff
@@ -2,6 +2,9 @@
 #define LIGHTNING_DAEMON_INVOICE_H
 #include "config.h"
 #include <bitcoin/preimage.h>
+#include <ccan/crypto/sha256/sha256.h>
```

Comment:
Hmm, we still use the invoice stuff for payments, it's pretty generic.  Happy for you to make a copy in anticipation of deleting the old one...